packages/proxy/lib/http/index.ts

```diff
@@ -268,6 +275,8 @@ export class Http {
   }
 
   reset () {
+    // TODO: when to reset marked origins?
+    // this.renderedHTMLOrigins = {}
```

Comment:
yes, when? this `Http` object's lifetime lasts for the entire time a project is open

Reply:
461e15c28

Reply:
461e15c28